dashboard/new-dashboard/src/components/bisect/BisectLauncher.vue

```diff
@@ -161,6 +173,7 @@ const model = reactive({
   excludedCommits: "",
   buildId,
   className,
+  targetJpsCompile: false,
```

Comment:
Do we need here the same conditions as in `BisectDialog.vue` instead of always `false`?